README.md

```diff
@@ -52,11 +52,14 @@ fastify.register(mail, pov: { {engine: { TEMPLATE_ENGINE_OBJECT } }, transporter
 
 // setup test route
 fastify.get("/sendmail", async (req, reply) => {
-  const recipients = ["test@example.com"]
-  const templates = "path/to/my/templates/"
-  const context = { name: "Test Name" }
 
-  const queued = await fastify.mail.sendMail(recipients, templates, context)
+  const opts = {
+    templatePath: "path/to/my/templates/",
+    context: { name: "Test Name" }
+  }
+
+  const queued = await fastify.mail.sendMail(message, opts)
```

Comment:
i missed it before, but we need a nodemailer `message` definition (our code example would have a syntax error without it!) :)  let's do it with a "to", "bcc", and a "from".

Reply:
added